mlir/include/mlir/Dialect/LLVMIR/LLVMAttrDefs.td

```diff
@@ -1232,6 +1232,47 @@ def LLVM_TBAATagArrayAttr
   let constBuilderCall = ?;
 }
 
+//===----------------------------------------------------------------------===//
+// Memory Model Relaxation Annations (mmra) Attributes
+//===----------------------------------------------------------------------===//
```

Comment:
nit: If we end up with only one attribute one banner comment should be good enough?